dist/rpm/apptainer.spec.in

```diff
@@ -132,27 +132,27 @@ mkdir -p $RPM_BUILD_ROOT%{_mandir}/man1
 make -C builddir DESTDIR=$RPM_BUILD_ROOT install man
 
 %files
-%attr(4755, root, root) %{_libexecdir}/singularity/bin/starter-suid
-%{_bindir}/singularity
-%{_bindir}/run-singularity
-%dir %{_libexecdir}/singularity
-%{_libexecdir}/singularity/bin/starter
-%{_libexecdir}/singularity/cni/*
-%dir %{_sysconfdir}/singularity
-%config(noreplace) %{_sysconfdir}/singularity/*.conf
-%config(noreplace) %{_sysconfdir}/singularity/*.toml
-%config(noreplace) %{_sysconfdir}/singularity/*.json
-%config(noreplace) %{_sysconfdir}/singularity/*.yaml
-%config(noreplace) %{_sysconfdir}/singularity/global-pgp-public
-%config(noreplace) %{_sysconfdir}/singularity/cgroups/*
-%config(noreplace) %{_sysconfdir}/singularity/network/*
-%config(noreplace) %{_sysconfdir}/singularity/seccomp-profiles/*
+%attr(4755, root, root) %{_libexecdir}/apptainer/bin/starter-suid
+%{_bindir}/apptainer
+%{_bindir}/run-apptainer
```

Comment:
we need to keep `run-singularity` here otherwise it would break the feature of directly executing a SIF image

Reply:
is this an external dependency?   breaking change for existing scripts owned by users?